docs/content/configuration/broker.md

```diff
@@ -37,6 +37,7 @@ Druid uses Jetty to serve HTTP requests.
 |`druid.server.http.numThreads`|Number of threads for HTTP requests.|10|
 |`druid.server.http.maxIdleTime`|The Jetty max idle time for a connection.|PT5m|
 |`druid.server.http.defaultQueryTimeout`|Query timeout in millis, beyond which unfinished queries will be cancelled|300000|
+|`druid.server.http.maxScatterGatherBytes`|Maximum number of bytes gathered from data nodes such as historicals and realtime processes to execute a query. This is an advance configuration that allows to protect in case broker is under heavy load and not utilizing the data gathered in memory fast enough and leading to OOMs. This limit can be further reduced at query time using `maxScatterGatherBytes` in the context. Note that having large limit is not necessarily bad if broker is never under heavy concurrent load in which case data gathered is processed quickly and freeing up the memory used.|Long.MAX_VALUE|
```

Comment:
I wonder why this property is prefixed by `druid.server` instead of `druid.broker`. Is it planned to be applied to other node types?

Reply:
i don't think it makes sense for any other node type. And now that you mention it, I think probably made more sense to call it `druid.broker.http.maxScatterGatherBytes` . we can possibly change it in future or remove this config if backpressure story improves.

Reply:
Thanks. It sounds good.